src/scss/components/optional/_overflow-carousel.scss

```diff
@@ -22,7 +22,8 @@
 
         .c-overflowCarousel-inner {
             @include media('<mid') {
-                overflow: scroll;
+                overflow-x: scroll;
+                overflow-y: hidden;
```

Comment:
Out of interested – if you didn't specify `overflow-y: hidden;`, would this still work? (as it wouldn't be `scroll` by default if not set, it'd be `visible`)

Reply:
just tested and removing `overflow-y: hidden;` re-creates the issue 😢

Reply:
Weird. CSS… :smil